omod/src/main/resources/config.xml

```diff
@@ -27,6 +27,7 @@
 		<aware_of_module>org.openmrs.module.legacyui</aware_of_module>
 		<aware_of_module>org.openmrs.module.owa</aware_of_module>
 		<aware_of_module>org.openmrs.module.cql</aware_of_module>
+		<aware_of_module>org.openmrs.module.fhir2</aware_of_module>
```

Comment:
We also need to make the FHIR sub module a condition resource like [this](https://github.com/openmrs/openmrs-module-fhir2/blob/de31a6838324ba2546b6eb71a69cdd77f36aa168/omod/src/main/resources/config.xml#L32) so it only gets loaded if FHIR2 is available.

Reply:
Note, too that the FHIR2 module version should probably be listed here.

Reply:
`<conditionalResource>
			<path>/lib/patientflags-fhir.*</path>
			<modules>
				<module>
					<moduleId>fhir2</moduleId>
					<version>1.9.0</version>
				</module>
			</modules>
		</conditionalResource>
Is this correct ?

Reply:
Yes, that looks right to me.